src/main/java/Controller/DukeController.java

```diff
@@ -0,0 +1,186 @@
+package Controller;
+import Tasks.*;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class DukeController {
+    private Parser parse = new Parser();
+    public DukeController() {};
+
+    public void printTask(ArrayList<Task> tasks, int count) {
+        System.out.println("--------------------------------------------");
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks.get(count).getPrintedLine());
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        System.out.println("--------------------------------------------");
+    }
+
+    public void printList(ArrayList<Task> tasks) {
+        int count = 1;
+        System.out.println("--------------------------------------------");
+        System.out.println("Here are the tasks in your list:");
+        for (Task task: tasks) {
+            System.out.println(count+ ". " + task.getPrintedLine());
+            count++;
+        }
+        System.out.println("--------------------------------------------");
+    }
+
+    public void printDone(ArrayList<Task> tasks, String in) {
+        int numerate;
+        numerate = parse.charNumber(in);
+        try {
+            if (numerate > tasks.size() || numerate == 0) {
+                throw new NullPointerException("There is no such event in your list");
+            }
+            System.out.println("--------------------------------------------");
+            tasks.get(numerate-1).markAsDone();
+            System.out.println(tasks.get(numerate-1).getPrintedLine());
+            System.out.println("--------------------------------------------");
+        }
+        catch (NullPointerException e) {
+            System.out.println(e.getMessage());
+            System.out.println("--------------------------------------------");
+        }
+    }
+
+    public boolean printTodo(ArrayList<Task> tasks, String in, String[] strings, int count1) {
+        boolean Error = false;
```

Comment:
boolean variables should be named to sound like boolean. Refer to coding standards.